Gaijinizer patch/Map461.txt

```diff
@@ -4,383 +4,480 @@
 
 いくら余やルカの出力が弱まっているとはいえ、これは…
-> CONTEXT: Map461/events/4/pages/0/223/Dialogue/alice_fc5 < UNTRANSLATED
+> CONTEXT: Map461/events/4/pages/0/223/Dialogue/alice_fc5
+Hold on, were these guys always this strong?!
 
+Even if Luka and I have had our powers weakened,
+this is absurd...#RonLocalization
 > END STRING
 
 > BEGIN STRING
 言ったでしょう……？\\C[10]『世界融解』\\C[0]……
 
 アバドーンに加担した際の特典のようなモノらしいけど
 これがある限り、あなたたちに負ける気がしないわ…♡
-> CONTEXT: Map461/events/4/pages/0/228/Dialogue/beelzebub_fc1 < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/228/Dialogue/beelzebub_fc1
+Didn't I tell you? It's the \\C[10]World Fusion\\C[0]...
+It's like a special perk for those who choose to
+work with Abaddon. And as long as we have it, I
+don't think you have any chance of beating us.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 この力を存分に使い、人々を恐怖に陥れ
 思いのままに繁殖する……
 
 それがアバドーンが私たちに提示した提案……
-> CONTEXT: Map461/events/4/pages/0/233/Dialogue/beelzebub_fc2 < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/233/Dialogue/beelzebub_fc2
+With free reign to use this power as we please,
+we'll plunge humanity into a state of terror and
+multiply as much as we desire. That is the future
+Abaddon offered to us.#Ron
 > END STRING
 
 > BEGIN STRING
 そして最優先は、\\N[1]……
 あなたを徹底的に搾って、無力化すること……♡♡
 
 うふふふふっ…♡
-> CONTEXT: Map461/events/4/pages/0/238/Dialogue/beelzebub_fc2 < UNTRANSLATED
+> CONTEXT: Map461/events/4/pages/0/238/Dialogue/beelzebub_fc2
+And so our top priority is you, \\N[1]...
 
+We'll thoroughly squeeze you out and render you
+powerless. Ufufufu... ♡♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 ………そろそろ、時間……
-> CONTEXT: Map461/events/4/pages/0/246/Dialogue/beelzebub_fc3 < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/246/Dialogue/beelzebub_fc3
+We're out of time...#Ron
 > END STRING
 
 > BEGIN STRING
 …………そうね。
 
 ルカ、\\N[1]。
 あなたたちとの決着は、次の機会にお預けよ。
-> CONTEXT: Map461/events/4/pages/0/248/Dialogue/beelzebub_fc1 < UNTRANSLATED
+> CONTEXT: Map461/events/4/pages/0/248/Dialogue/beelzebub_fc1
+...You're right.
 
+Luka, \\N[1]... we'll have to put this on
+hold and settle things next time we meet.#Ron
 > END STRING
 
 > BEGIN STRING
 本当はもう少し遊んでいたいけど……
 
 この世界はもう『時間切れ』みたい……
-> CONTEXT: Map461/events/4/pages/0/253/Dialogue/beelzebub_fc1 < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/253/Dialogue/beelzebub_fc1
+I really would have liked to play with you a
+little longer, but it seems this world is already
+out of time...#Ron
 > END STRING
 
 > BEGIN STRING
 なに…！？
-> CONTEXT: Map461/events/4/pages/0/257/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/257/Dialogue/kazuya
+What?!#Ron
 > END STRING
 
 > BEGIN STRING
 この世界も、じきアバドーンに喰われる………
 
 そして『世界融解』の力は更に強まり
 私たちの力は、もっと強くなるのよ…
-> CONTEXT: Map461/events/4/pages/0/278/Dialogue/beelzebub_fc1 < UNTRANSLATED
-
+> CONTEXT: Map461/events/4/pages/0/278/Dialogue/beelzebub_fc1
+This world's going to be consumed by Abaddon
+soon. That will further strengthen the World
+Fusion, making us even more powerful.#Ron
 > END STRING
```

Comment:
Minor revisions for flow:
```
This world's going to be consumed by Abaddon
soon. It will further strengthen the World
Fusion, making us even more powerful...
```